cohttp-eio/src/writer.ml

```diff
@@ -0,0 +1,49 @@
+(* Original https://github.com/talex5/httpaf/blob/eio/lib/serialize.ml *)
```

Comment:
If this comes from somewhere else, we need to retain the copyright header:
```
https://github.com/talex5/httpaf/blob/eio/lib/serialize.ml#L1-L32
```

Reply:
Added copyright notice.